### ISSUE TITLE
Validation Accuracy Not Improving for tuberculosis

### ISSUE THREAD

**Issue by mohitthakur007** [jqug/microscopy-object-detection#3]
I was trying to replicate the tuberculosis model but seems like the dataset is changed as logs of the .ipynb file do not match. 
IMAGE OF NEWLY AVAILABLE DATASET
![Screenshot 2020-10-28 at 11 47 02 PM](https://user-images.githubusercontent.com/41528050/97480364-4e47d600-1979-11eb-8ad6-5ca770e613c7.png)
IMAGE OF LOGS AVAILAILABLE IN THIS REPO
![Screenshot 2020-10-29 at 12 01 38 AM](https://user-images.githubusercontent.com/41528050/97480834-fa89bc80-1979-11eb-8f45-fabdc6ee494d.png)

I still tried to go ahead with training but after a few epochs the validation accuracy is stuck at 0.55492.
IMAGE of model validation accuracy stuck at 55% 
![Screenshot 2020-10-28 at 11 49 19 PM](https://user-images.githubusercontent.com/41528050/97480663-bd252f00-1979-11eb-8419-ee9baabc0a99.png)

**If the dataset is changed, can you please provide me original dataset ?
If not, can you please advice me what kind of hyper-parameters should be tuned to avoid this issue?**
P.S. I'm attaching a couple of graphs and images which might help you to understand my issue better.
![WhatsApp Image 2020-10-29 at 00 10 47](https://user-images.githubusercontent.com/41528050/97483053-fd39e100-197c-11eb-81cd-be511b820c82.jpeg)

![WhatsApp Image 2020-10-29 at 00 10 06](https://user-images.githubusercontent.com/41528050/97483071-0165fe80-197d-11eb-9b6e-e38035dc5299.jpeg)

The red boxes around the bacteria is also not visible
![WhatsApp Image 2020-10-29 at 00 10 05](https://user-images.githubusercontent.com/41528050/97483080-062ab280-197d-11eb-953a-c1ca153a7452.jpeg)

Please try and help, I been stuck for quite long. Thanks in advance.
